labgrid/driver/power/netio_json.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Add HTTP-JSON-API functions for labgrid-netio power driver.
+Set model in YML-CFG to 'netio_json'.
+"""
+__author__ = "Eugen Wiens, Christian Happ, Raffael Krakau"
+__copyright__ = "Copyright 2023, JUMO GmbH & Co. KG"
+__email__ = "Eugen.Wiens@jumo.net, Christian.Happ@jumo.net"
+__date__ = "2023-07-05"
+__status__ = "Production"
+
+import json
+
+import requests
+
+PORT = 80
+
+
+class __NetioControl:
+    """
+    Netio-Class to switch sockets via HTTP-JSON-API
+    """
+    __host: str
```

Comment:
`__host` and the variables below are defined as class variables, however they are not used anywhere? Every place uses the instance variables instead.

Reply:
`__host` is a private copy of `host` (added to prevent side-effects)
At line 105 this copy will be used.
`__host` will be mangled and show up as `NetioControl__host` which prevents any other operation from accidentally modifying it.

Reply:
Can this be resolved?

Reply:
Any further suggestion on how to handle this preferably?